tests/backstop_data/Readme.md

```diff
@@ -0,0 +1,58 @@
+# Covered Screens
+The backstop test suite for Civicase 5.1 extension covers following screens
+
+## Dashboard
+- [ ] Dashboard Main screen - With overview table expanded and tooltip visible on one of the titles
+- [ ] Dashboard Overview table with gear icon opened
+- [ ] Dashboard Main screen - with loading screens
+- [ ] Dashboard Main screen - Calendar - Acitivity card
+- [ ] Dashboard Main screen  - Empty State
+
+## Activities Feed Panel
+- [ ] Activities Feed Panel - Main screen
+- [ ] Activities Feed Panel - Loading screen
+- [ ] Activities Feed Panel - Bulk action Checkbox enabled and one checkbox checked, and bulk action dropdown opened
+- [ ] Activities Feed Panel - Load more state
+- [ ] Activities Feed Panel -  filter dropdowns
+- [ ] Activities Feed Panel - with one filter enabled
+- [ ] Activities Feed Panel - one activity selected
+- [ ] Activities Feed Panel - Activity card menu on case overview
+- [ ] Activities Feed Panel - Empty State
+- [ ] Activities Feed Panel - Detail - Edit state
+- [ ] Activities Feed Panel - Detail - Delete state
+- [ ] Activities Feed Panel - Under Manage Cases
+- [ ] Activities Feed Panel - Under Contact Page
+
+## Manage Cases Screens
+- [ ] Manage Cases List - Main screen
+- [ ] Manage Cases List - Other Criterion filter button
+- [ ] Manage Cases List - Loading screen
+- [ ] Manage Cases List - selected case
+- [ ] Manage Cases List - Case Overview - with drawer closed
+- [ ] Manage Cases List - Case Overview - Empty States
+- [ ] Manage Cases List - Case Overview - with calendar activity opened
+- [ ] Manage Cases List - Case Overview - Loading
+- [ ] Manage Cases List - Case Overview - Edit custom Data
+- [ ] Manage Cases List - Case Overview - Add new - Open the popup
+- [ ] Manage Cases List - People - Case Roles
+- [ ] Manage Cases List - People - Case Roles - Loading
+- [ ] Manage Cases List - People - Other Relationships
+- [ ] Manage Cases List - People - Other Relationships - Loading
+- [ ] Manage Cases List - Files
+- [ ] Manage Cases List - Files - Loading
+- [ ] Manage Cases List - Files - Upload files - *Skip this if is hard to implement*
+
+## Contact Page
+- [ ] Contact Page - Case Tab
+- [ ] Contact Page - Case Tab - Loading screen
+- [ ] Contact Page - Case Tab - Loading more results icon (When clicked on load more
+
+## Modals
+- [ ] Modals - Contact Popover Modal
+- [ ] Modals - + 1 Contact Popover Modal
```

Comment:
Instead of `+ 1`, we can use `Additional Contacts Popover`.
Also isn't `Popover` and `Modal` the same thing? we should use either of them